mozetl/taar/taar_amowhitelist.py

```diff
@@ -0,0 +1,86 @@
+#!/bin/env python
+
+import click
+import json
+import logging
+import logging.config
+from dateutil.parser import parse
+from tempfile import NamedTemporaryFile
+import datetime
+
+from taar_utils import read_from_s3, store_json_to_s3
+
+AMO_DUMP_BUCKET = 'telemetry-parquet'
+AMO_DUMP_PREFIX = 'telemetry-ml/addon_recommender/'
+
+AMO_DUMP_BASE_FILENAME = 'extended_addons_database'
+FILTERED_AMO_BASE_FILENAME = 'whitelist_addons_database'
+
+AMO_DUMP_FILENAME = AMO_DUMP_BASE_FILENAME + '.json'
+FILTERED_AMO_FILENAME = FILTERED_AMO_BASE_FILENAME + '.json'
+
+MIN_RATING = 3.0
+MIN_AGE = 60
+
+logger = logging.getLogger('amo_whitelist')
+
+
+class AMOTransformer:
```

Comment:
Can we please add a documentation for this? What is it supposed to do? For which purpose?

Reply:
Docs added